.npmrc

```diff
@@ -0,0 +1,3 @@
+username=${NPM_USERNAME}
+email=${NPM_EMAIL}
+//registry.npmjs.org/:_authToken=${AUTH_TOKEN}
```

Comment:
Eliminates the need for another script to generate this file.

Reply:
does this need to be added to one of the `.ignore` files?

Reply:
I don't _think_ so.  We don't want it in `.gitignore` since it's a versioned resource.  The `.eslintignore` shouldn't matter because we're only linting in the `src` and `scripts` directories.  It would seem strange if `.npmignore` ignored `.npmrc`.